events/importer/turku_old_events.py

```diff
@@ -378,21 +369,41 @@ def _import_event(self, lang, event_el, events, event_type):
             if location_extra_info.strip().endswith(','):
                 location_extra_info = location_extra_info.strip()[:-1]
 
+            # Define location_extra_info dict.
             evItem['location_extra_info'] = {
-                "fi": location_extra_info if location_extra_info else None,
-                "sv": location_extra_info if location_extra_info else None,
-                "en": location_extra_info if location_extra_info else None
+                "fi": str(eventTku['address'])+' / '+location_extra_info if location_extra_info else eventTku['address'],
```

Comment:
Use string formatting: `'%(address)s/%(extra)s'`

Reply:
Thank you, that is good practice! 
I went over some documentation yesterday and found something.
I think I will use the `'{first} {last}'.format(**data)` approach though since it's newer: https://www.python.org/dev/peps/pep-3101/ goes over the % and string.format() differences.

Reply:
Well I now understand why % would be better. String.format is really tricky when it comes to None values.